ci/long_running_tests/config.yaml

```diff
@@ -14,48 +14,35 @@ auth:
 
 head_node:
     InstanceType: m5.xlarge
-    ImageId: ami-0def3275  # Default Ubuntu 16.04 AMI.
-
-    # Set primary volume to 25 GiB
+    ImageId: ami-0888a3b5189309429  # DLAMI
     BlockDeviceMappings:
         - DeviceName: /dev/sda1
           Ebs:
-              VolumeSize: 50
+              VolumeSize: 150
 
 worker_nodes:
     InstanceType: m5.large
-    ImageId: ami-0def3275  # Default Ubuntu 16.04 AMI.
-
-    # Set primary volume to 25 GiB
+    ImageId: ami-0888a3b5189309429  # DLAMI
     BlockDeviceMappings:
         - DeviceName: /dev/sda1
           Ebs:
-              VolumeSize: 50
+              VolumeSize: 150
 
     # Run workers on spot by default. Comment this out to use on-demand.
     InstanceMarketOptions:
         MarketType: spot
 
 # List of shell commands to run to set up nodes.
 setup_commands:
-    - sudo apt-get update
-    - sudo apt-get install -y build-essential curl unzip
-    # Install Anaconda.
-    - wget https://repo.continuum.io/archive/Anaconda3-5.0.1-Linux-x86_64.sh || true
-    - bash Anaconda3-5.0.1-Linux-x86_64.sh -b -p $HOME/anaconda3 || true
-    - echo 'export PATH="$HOME/anaconda3/bin:$PATH"' >> ~/.bashrc
-    - echo 'termcapinfo xterm* ti@:te@' >> ~/.screenrc
```

Comment:
@robertnishihara 	what was this for?

Reply:
Good question, I don't remember adding that.

Reply:
It's doesn't work for tmux anyways :(

For some reason, tmux/screen require these arcane configs to enable mouse scrolling.